ecdsa/CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.17.0 (UNRELEASED)
+
+### Changed
+- `DigestPrimitive` was removed from hazmat ([#945])
```

Comment:
Perhaps put the removal under `### Removed`

Reply:
Well, it was not removed per se. I changed the wording of the line instead.